inst/ms/aggreCAT.qmd

```diff
@@ -400,39 +423,43 @@ for a single claim to illustrate this structure of the core
 `data_ratings` dataset.
 
 ```{r}
-#| label: data_ratings
+#| label: data_ratings-sample
 #| message: false
 #| prompt: true
+#| results: hold
+
 library(tidyverse,quietly = TRUE)
 library(aggreCAT)
 data(data_ratings)
 data_ratings %>% 
-  print(n = 18)
+  dplyr::filter(paper_id == dplyr::first(paper_id), 
+                user_name == dplyr::first(user_name)) %>% 
+  print(., n = nrow(.))
 ```
 
 Not all data necessary for constructing weights on performance is
 contained in `data_ratings`. Additional data collected as part of the
 repliCATS IDEA protocol are contained within separate datasets to
-`data_ratings`. Justifications for giving particular judgements are
-contained in `data_justifications`. on the repliCATS platform users were
-given the option to comment on others' justifications (`data_comments`),
-to vote on others' comments (`data_comment_ratings`) and on others'
-justifications (`data_justification_ratings`). Finally, [aggreCAT]{.pkg}
-contains three 'supplementary' datasets containing data collected
-externally to the repliCATS IDEA protocol: `data_supp_quiz`,
+`data_ratings`. On the repliCATS platform users were given the option to
+comment on others' justifications (`data_comments`), to vote on others'
+comments (`data_comment_ratings`) and on others' justifications
+(`data_justification_ratings`). Justifications for giving particular
+judgements are contained in `data_justifications`. Finally,
+[aggreCAT]{.pkg} contains three 'supplementary' datasets containing data
+collected externally to the repliCATS IDEA protocol: `data_supp_quiz`,
 `data_supp_priors`, and `data_supp_reasons`.
 
 #### Quiz Score Data {#sec-quiz-supplementary-data}
 
 Prior to the workshop, participants also completed an optional quiz on
 statistical concepts and meta-research that we expect participants to be
 aware of in order to reliably evaluate the replicability of research
-claims. Quiz responses are contained in `data_supp_quiz` and are
-used to construct performance weights for the aggregation method
-`QuizWAgg` where each participant receives a `quiz_score` from 0 - **X
-(TODO)** if they completed the quiz, and `NA` if they did not attempt or
-fully complete the quiz [see @Hanea2021 for further details]. (Question
-for Bonnie, possibly Rose?: Pretty sure they get points for any question
+claims. Quiz responses are contained in `data_supp_quiz` and are used to
+construct performance weights for the aggregation method `QuizWAgg`
+where each participant receives a `quiz_score` from 0 - **X (TODO)** if
```

Comment:
X depends on which of the quiz columns are used. The one we use for QuizWAgg goes up to 16, the one giving equal weights to all questions is 22, and the stats portion only is IIRC 12.

Reply:
Thanks David, I think for the sake of simplicity in the paper, I will omit the specific score range, because the details are not actually relevant given that we are just demonstrating how to use your own dataset to construct weights.